IPs/RSKIPxxx.md

```diff
@@ -0,0 +1,86 @@
+# Bridge UTXO set size management
+
+
+|RSKIP          | xxx |
+| :------------ |:-------------|
+|**Title**      |Bridge UTXO set size management|
+|**Created**    |AUG-2021 |
+|**Author**     | SDL |
+|**Purpose**    |Sec, Usa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+#  **Abstract**
+
+One of the problems of the RSK bridge is that it doesn't have any means to prevent the proliferation and fragmentation of UTXOs. This can lead to a number of cost, usability, efficiency and security problems defined in [RSKIP265](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP265.md). This RSKIP is part of a series of RSKIPs that address these problems (RSKIP265 to RSKIP268). In particular, this RSKIP proposes a new protocol for the Bridge to shrink or expand the UTXOs to keep a UTXO set of a predefined size. This is done by splitting amounts (expansion) or joining amounts (consolidation). Lower bound and upper bound thresholds are used to avoid frequent UTXO management operations. 
```

Comment:
> a series of RSKIPs that address these problems (RSKIP265 to RSKIP268)
Shouldn't the *series* be from 270-273?

Reply:
> UTXO set of a predefined size
 
size ->  range?  e.g. ... to keep the size of the UTXO set within a predetermined range.